ios/Classes/SwiftWorkmanagerPlugin.swift

```diff
@@ -63,19 +63,12 @@ extension SwiftWorkmanagerPlugin {
     
     override public func application(_ application: UIApplication, performFetchWithCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) -> Bool {
         
-        // First, let's retrieve our callBack handle
         guard let callbackHandle: Int64 = getStoredCallbackHandle() else {
-            let errorMessage = "[\(#file)] Could not start the Flutter engine : no stored callback handle."
-            if #available(iOS 10.0, *) {
-                os_log("%@", errorMessage)
-            } else {
-                NSLog(errorMessage)
-            }
+            logError("[\(String(describing: self))] Could not start the Flutter engine : no stored callback handle.")
```

Comment:
We have a very similar[ error message](https://github.com/vrtdev/flutter_workmanager/blob/master/android/src/main/kotlin/be/tramckrijte/workmanager/WorkmanagerCallHandler.kt#L42). We send it back to the user.

```
result.error(
    "1",
    "You have not properly initialized the Flutter WorkManager Package. " +
            "You should ensure you have called the 'initialize' function first! " +
            "Example: \n" +
            "\n" +
            "`Workmanager.initialize(\n" +
            "  callbackDispatcher,\n" +
            " )`" +
            "\n" +
            "\n" +
            "The `callbackDispatcher` is a top level function. See example in repository.",
    null
)
```

Reply:
Interesting, at that point in time we are in the `AppDelegate`s `performFetchWithCompletionHandler` though and don't have a `MethodChannel.Result` to post the error on. This is an extra safeguard right before the `iOSPerformFetch` method will invoked

Reply:
No problem, maybe show the same error message on both platforms?